nasa-gibs/worldview
# WV-2829 Outage Notifications

## Description
When we have "outage" notifications posted in Worldview, these should pop up in their own window as soon as you load Worldview, using the existing notification window. The "Welcome to Worldview" tour story window should only load after you close the outage notification.

Fixes # wv-2829

## How To Test

- git checkout wv-2829-outages
- npm ci
- npm run watch
- Open local WV page(s) ([example 1](http://localhost:3000/?v=-110.73131735450241,-44.68966273806126,1.2873429044083622,32.04705320375335&l=Reference_Labels_15m(hidden),Reference_Features_15m(hidden),Coastlines_15m,MODIS_Aqua_Cloud_Phase_Optical_Properties(disabled=3-4),MODIS_Terra_CorrectedReflectance_TrueColor&lg=true&t=2020-09-28-T19%3A16%3A13Z), [example2](http://localhost:3000/?v=-6362614.836762745,-4125453.3768259212,6536465.756444529,4710869.795614285&p=arctic&l=Coastlines_15m,MODIS_Aqua_Cloud_Top_Height_Day,MODIS_Terra_CorrectedReflectance_TrueColor&lg=true&t=2022-08-28-T19%3A17%3A42Z)) and confirm that the notification window pops up on initial page load
- [Load WV with no parameters](http://localhost:3000/) & confirm that the Tour window is displayed after notifications are dismissed
- Run e2e tests & confirm that all tests are passing

## Merging

Please use the `squash and merge` commit method unless each commit in your branch is vital to the commit history of main.

@nasa-gibs/worldview
